docs/pages/access-controls/sso/oidc.mdx

```diff
@@ -193,6 +193,28 @@ spec:
        roles: [ "access" ]
 ```
 
+### Optional: Max Age
+
+Teleport supports setting the `max_age` field since version 13.3.7 to control the
+maximum age of users' sessions before they will be forced to reauthenticate. By
+default `max_age` is unset, meaning once a user authenticates using OIDC they will
+not have to reauthenticate unless the configured OIDC provider forces them to. This
+can be set to a duration of time to force users to reauthenticate more often. If
+`max_age` is set to zero seconds, users will be forced to reauthenticate every time.
```

Comment:
I think "forced to reauthenticate every time" can be more precise. Does it mean, "forced to reauthenticate with their OIDC provider every time they authenticate with Teleport"?

Reply:
That's correct, thanks